integration-tests/features/steps/gemini.py

```diff
@@ -166,8 +173,7 @@ def check_cves_for_epv(context, cves, p, v, e):
         package = check_and_get_attribute(dependency, "name")
         version = check_and_get_attribute(dependency, "version")
         if ecosystem == e and package == p and version == v:
-            assert int(cve_count) == int(cves), \
-                "{exp} CVEs expected, but {found} was found".format(exp=cves, found=cve_count)
+            check_cve_count(cve_count, cves)
```

Comment:
There could be a CVE found for EPV later. I guess we should check `CVEs atleast expected but found less` statements

Reply:
yes @miteshvp, let's change it. TY